### PR TITLE
Migrate to latest Oculus SDK (0.8)

### PR DESCRIPTION
**Pull request by jherico**
This PR also includes the changes in https://github.com/highfidelity/hifi/pull/6598

### CODE REVIEW

File: libraries/display-plugins/CMakeLists.txt [hunk 1]
```diff
@@ -8,7 +8,8 @@ GroupSources("src/display-plugins")
 
 target_oglplus()
 
-if (WIN32)
+#if (WIN32)
+if (FALSE)
```
Comment (ZappoMan) [inline libraries/display-plugins/CMakeLists.txt]: did you mean to leave this as FALSE?

File: libraries/input-plugins/src/input-plugins/ViveControllerManager.cpp [hunk 2]
```diff
@@ -26,6 +26,8 @@
 
 #include <controllers/StandardControls.h>
 
+#undef Q_OS_WIN
```
Comment (ZappoMan) [inline libraries/input-plugins/src/input-plugins/ViveControllerManager.cpp]: maybe we should add a comment explaining that you did this intentionally?

**Comment by hifi-gustavo** [comment]
Build console output for commit: f92028c1745c9fbba90fc0ab2d696d666a5b78af available at: https://gist.github.com/e30db5487cc5ef91f9c8<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-6595.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-6595.dmg

File: libraries/display-plugins/CMakeLists.txt (hunk 1 above)
Comment (jherico) [inline libraries/display-plugins/CMakeLists.txt]: Yes.  For some reason, when the OpenVR SDK is linked against the application, whether it's activated or not, it breaks the Oculus SDK somehow.

File: libraries/input-plugins/src/input-plugins/ViveControllerManager.cpp (hunk 2 above)
Comment (jherico) [inline libraries/input-plugins/src/input-plugins/ViveControllerManager.cpp]: fixed

**Comment by hifi-gustavo** [comment]
Build console output for commit: eac185c80ffbaeb502d33f6deba69a79ce5a45dc available at: https://gist.github.com/e30db5487cc5ef91f9c8<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-6595.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-6595.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: f92028c1745c9fbba90fc0ab2d696d666a5b78af available at: https://gist.github.com/e30db5487cc5ef91f9c8<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-6595.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-6595.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 581c8f59b76e1c5bd8e4ee3bb4d95347df8dbf18 available at: https://gist.github.com/e30db5487cc5ef91f9c8<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-6595.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-6595.dmg

**Comment by jherico** [comment]
test this please

**Comment by hifi-gustavo** [comment]
Build console output for commit: 581c8f59b76e1c5bd8e4ee3bb4d95347df8dbf18 available at: https://gist.github.com/e30db5487cc5ef91f9c8<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/High+Fidelity-PR-6595-win32.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-6595.dmg

File: libraries/display-plugins/CMakeLists.txt (hunk 1 above)
Comment (jherico) [inline libraries/display-plugins/CMakeLists.txt]: I could actually remove the OpenVR disabling commit entirely, but only if you want to make sure https://github.com/highfidelity/hifi/pull/6598 gets merged before this

**Comment by hifi-gustavo** [comment]
Build console output for commit: 5d4231b683e292832122b1a5b7d27a4ff547cebd available at: https://gist.github.com/e30db5487cc5ef91f9c8<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/High+Fidelity-PR-6595-win32.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-6595.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: e3dd938d12ac9a3591df74c379e80470e63618a4 available at: https://gist.github.com/e30db5487cc5ef91f9c8<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/High+Fidelity-PR-6595-win32.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-6595.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: d58c734b9f918eb1fc509d082b2db9457353bd59 available at: https://gist.github.com/e30db5487cc5ef91f9c8<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/High+Fidelity-PR-6595-win32.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-6595.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 1644bda9080983237524dea8d5db6636976af1aa available at: https://gist.github.com/e30db5487cc5ef91f9c8<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/High+Fidelity-PR-6595-win32.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-6595.dmg

**Pull request merged**